### ISSUE TITLE
Default Clojure source file for new console app project

### ISSUE THREAD

**Issue by frankhale** [vsClojure/vsClojure#112]
I'd like to see the default Clojure source file for a new console app project be a little bit more useful.

For instance I think the main function should look like this:

``` clojure
(defn -main [& args]
  (println "Hello, World!")
  (println "Press any to quit...")
  (System.Console/ReadKey))
```
- Added Hello, World
- Change to read any key
- The last paren is _not_ on a new line

**Comment by speige** [comment]
I think for a command-line project, that is the default template. Maybe we should change the class library template to use a gen-class template?

There are templates available when you right-click/"add new item" from the .cljproj project, maybe we can copy one of those?

**Comment by frankhale** [comment]
Sorry, I should have said console app. Darn it!

**Comment by frankhale** [comment]
That isn't the default for a command line app, I reworked the default source to be what I pasted above. The default one looks like this:

``` clojure
(ns program
  (:gen-class))

(defn -main [& args]
  (println "Press enter to quit...")
  (read-line)
)
```

Additionally I think (:gen-class) should probably be removed. I'm not sure how many people would need that for just a console app. I think it'd be rare to want to reference that Clojure code from another project within the solution for a console app. For a class library I think it should be there.
